client/src/modules/AutoTest/components/SelfEducation/SelfEducation.tsx

```diff
@@ -101,16 +103,24 @@ function SelfEducation({ courseTask }: SelfEducationProps) {
                           answer
                         )}
                       </Radio>
-                    </Row>
-                  ))}
+                    ))}
+                  </Space>
                 </Radio.Group>
               )}
             </Form.Item>
           );
         },
       )}
+      <style jsx>{styles}</style>
     </>
   );
 }
 
+const styles = css`
+  :global(.ant-radio) {
```

Comment:
you apply these styles to each radio button on the page, even the one, which is not used in that component(ie real global styles). Please avoid doing that. For margin you can add additional wrapper, for align you can add either scoping or override inline styles

Reply:
If I do it this way:

```diff
const styles = css`
--  :global(.ant-radio) {
++  .self-education :global(.ant-radio) {
    align-self: flex-start !important;
    margin-top: 3px !important;
  }
`;
```

will it be OK?

Reply:
yes, that would be better, thanks